setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='django-multidb-router',
-    version='0.6',
+    version='0.7',
```

Comment:
Sorry, this was to force pip to install my changes as I had no control over the server.